src/lib/logger/logger.mli

```diff
@@ -103,6 +107,24 @@ module Transport : sig
 
   val create : (module S with type t = 'transport_data) -> 'transport_data -> t
 
+  module File_system : sig
```

Comment:
Delete

Reply:
didn't you write this? what was this for again